README.md

```diff
@@ -20,6 +51,14 @@ For **Serverless-Offline**:
 
 2. Use the given ROUTE on **Postman**, eg. http://localhost:3000/2015-03-31/functions/census-dev-hello/invocations to Test appropriately
 
+### Dependencies
+You are likely need to run the following commands to download necessary packages:
+
+```
+npm install -g serverless
+npm install -g mocha
```

Comment:
we can add this to the scripts section in npm. that way it'll automatically install when someone uses "npm install"

Reply:
ah, like in package.json to define a new command?